### PR TITLE
Created functionality for request form

### PR DESCRIPTION
**Pull request by Ruslana48**

### CODE REVIEW

**Comment by R-accoo-n** [comment]
|File|Coverage [26.45%]|:green_apple:|
|:-|:-:|:-:|
|[WebSecurityConfig.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fsecurity%2FWebSecurityConfig.java)|100%|:green_apple:|
|[WebMvcConfig.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fsecurity%2FWebMvcConfig.java)|92.96%|:green_apple:|
|[FundraisingService.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fservices%2FFundraisingService.java)|34.48%|:green_apple:|
|[UserService.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fservices%2FUserService.java)|30.84%|:green_apple:|
|[FundraisingController.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fcontrollers%2FFundraisingController.java)|27.27%|:green_apple:|
|[RequestController.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fcontrollers%2FRequestController.java)|22.22%|:green_apple:|
|[AuthController.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fcontrollers%2FAuthController.java)|17.14%|:green_apple:|
|[User.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fmodels%2FUser.java)|13.51%|:green_apple:|
|[UserDto.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fdto%2FUserDto.java)|0%|:x:|
|[MilitaryDto.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fdto%2FMilitaryDto.java)|0%|:x:|
|[RequestDto.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/b9f3028d059c8afe31d57e1b9d246dcd84c4f466/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fdto%2FRequestDto.java)|0%|:x:|

|Total Project Coverage|24.01%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/nameless/volunteerproject/controllers/RequestController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import javax.validation.Valid;
+
+@Controller
+public class RequestController {
+    private FundraisingService fundraisingService;
+
+    public RequestController(FundraisingService fundraisingService) {
+        this.fundraisingService = fundraisingService;}
+
+    //ADD MILITARY ID
```
Comment (nazardyda) [inline src/main/java/com/nameless/volunteerproject/controllers/RequestController.java:21]: //TODO MILITARY ID

File: src/main/java/com/nameless/volunteerproject/controllers/AuthController.java [hunk 2]
```diff
@@ -18,18 +20,17 @@ public class AuthController {
     private UserService userService;
 
     public AuthController(UserService userService) {
```
Comment (nazardyda) [inline src/main/java/com/nameless/volunteerproject/controllers/AuthController.java:22]: Try to use @RequiredArgsConstructor

File: src/main/java/com/nameless/volunteerproject/controllers/FundraisingController.java [hunk 3]
```diff
@@ -13,8 +21,9 @@
 @RestController
 public class FundraisingController {
 
-    @Autowired
     private FundraisingService fundraisingService;
+    public FundraisingController(FundraisingService fundraisingService) {
```
Comment (nazardyda) [inline src/main/java/com/nameless/volunteerproject/controllers/FundraisingController.java:25]: Try to use @RequiredArgsConstructor

**Review by nazardyda — APPROVED** [review]

**Comment by R-accoo-n** [comment]
|File|Coverage [26.31%]|:green_apple:|
|:-|:-:|:-:|
|[WebSecurityConfig.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fsecurity%2FWebSecurityConfig.java)|100%|:green_apple:|
|[WebMvcConfig.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fsecurity%2FWebMvcConfig.java)|92.96%|:green_apple:|
|[FundraisingService.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fservices%2FFundraisingService.java)|34.48%|:green_apple:|
|[UserService.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fservices%2FUserService.java)|30.84%|:green_apple:|
|[RequestController.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fcontrollers%2FRequestController.java)|22.22%|:green_apple:|
|[AuthController.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fcontrollers%2FAuthController.java)|17.14%|:green_apple:|
|[FundraisingController.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fcontrollers%2FFundraisingController.java)|15.79%|:green_apple:|
|[User.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fmodels%2FUser.java)|13.51%|:green_apple:|
|[UserDto.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fdto%2FUserDto.java)|0%|:x:|
|[MilitaryDto.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fdto%2FMilitaryDto.java)|0%|:x:|
|[RequestDto.java](https://github.com/R-accoo-n/Nameless-volunteer-project/blob/6407d5f918ed08cdb64114ca27927cc56c90a3b8/src%2Fmain%2Fjava%2Fcom%2Fnameless%2Fvolunteerproject%2Fdto%2FRequestDto.java)|0%|:x:|

|Total Project Coverage|23.37%|:green_apple:|
|:-|:-:|:-:|

**Pull request merged**